tools/src/main/scala/scala/scalanative/tools/Config.scala

```diff
@@ -0,0 +1,80 @@
+package scala.scalanative
+package tools
+
+import scalanative.io.VirtualDirectory
+import nir.Global
+
+trait Config {
```

Comment:
You should seal that trait (and any other similar trait), otherwise you cannot add methods later on.